src/Microsoft.Framework.Runtime/NuGet/Authoring/ManifestReader.cs

```diff
@@ -189,28 +188,30 @@ public static List<ManifestReference> ReadReference(XElement referenceElement, b
             return references;
         }
 
-        private static List<ManifestFrameworkAssembly> ReadFrameworkAssemblies(XElement frameworkElement)
+        private static IEnumerable<FrameworkAssemblyReference> ReadFrameworkAssemblies(XElement frameworkElement)
         {
             if (!frameworkElement.HasElements)
             {
-                return new List<ManifestFrameworkAssembly>(0);
+                return new List<FrameworkAssemblyReference>(0);
             }
 
             return (from element in frameworkElement.ElementsNoNamespace("frameworkAssembly")
                     let assemblyNameAttribute = element.Attribute("assemblyName")
-                    where assemblyNameAttribute != null && !String.IsNullOrEmpty(assemblyNameAttribute.Value)
-                    select new ManifestFrameworkAssembly
-                    {
-                        AssemblyName = assemblyNameAttribute.Value.SafeTrim(),
-                        TargetFramework = element.GetOptionalAttributeValue("targetFramework").SafeTrim()
-                    }).ToList();
+                    where assemblyNameAttribute != null && !string.IsNullOrEmpty(assemblyNameAttribute.Value)
```

Comment:
nit:
`let assemblyNameAttribute = element.Attribute("assemblyName")
where assemblyNameAttribute != null && !string.IsNullOrEmpty(assemblyNameAttribute.Value)`

could be just:

`let assemblyName = (string)element.Attribute("assemblyName")
where !string.IsNullOrEmpty(assemblyName)`